accepted/2021/preview-features/preview-features.md

```diff
@@ -0,0 +1,418 @@
+# Preview Features
+
+**PM** [Immo Landwerth](https://github.com/terrajobst)
+
+Starting with .NET 5, we've moved to fixed release schedule whereby we ship one
+version of .NET once a year, at RTM quality, with every other year being a long
+term support (LTS) release.
+
+This is in stark contrast to the .NET Framework days where we had multiple years
+for a given release, typically two to three years.
+
+Shipping more frequently has many advantages, such as being able to react
+quicker to trends and having an organization and customer base that is used to
+shipping & absorbing technology quicker to make the most of today's fast moving
+world.
+
+With .NET Core, we've solved the problem of the extremely high in-place
+compatibility bar that .NET Framework has. This reduces risks for unintentional
+behavioral changes and also allows certain parts of .NET to make deliberate
+breaking changes if necessary.
+
+As a result, this makes it viable to innovate across runtime, libraries, and
+languages, allowing us to build more compelling features that would have been
+almost impossible to do on .NET Framework. Examples include static linking,
+default interface members, and ref structs (such as `Span<T>`).
+
+However, it's sometimes challenging to deliver cross-cutting feature work in a
+single release, especially when they are breaking new ground in terms of
+expressiveness. A good example of such a feature is the upcoming improved
```

Comment:
is expressiveness the right word? Or, could you explain what you mean by it?

Reply:
By "expressiveness" I mean being able to represent concepts that you couldn't do before. These are usually type system features (such as async/await, static interface members, nullability) as opposed to pure syntactic sugar (e.g. file scope namespace declarations, the using modifier for locals) or a few more types/methods in the BCL.

Reply:
The word confused me too. The sentence can probably just end after "However, it's sometimes challenging to deliver cross-cutting feature work in a single release" -- I don't think anybody will disagree with that!